### PR TITLE
passing tests and returning correct data

### PR DESCRIPTION
**Pull request by cruzang000**
api service class is ready

querying for trivia can be done like this:
`Trivia[] triviaSet = triviaApi.getTrivia(category, type, difficulty, amount);`
and will receive an array of Trivia Objects:
`[Trivia, Trivia, Trivia]`

querying for categories can be done like this:
`Category[] categories = triviaApi.getAllCategories(); `
and will receive an array of Category Objects:
`[Category, Category, Category]`

### CODE REVIEW

File: src/main/java/com/trivialTrivia/entity/Category.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.trivialTrivia.entity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
```
Comment (esajdak) [inline src/main/java/com/trivialTrivia/entity/Category.java:7]: Small change, but we could update this to import all from lombok and reduce 4 lines down to 1

File: src/main/test/com/trivialTrivia/api/TriviaApiTest.java [hunk 2]
```diff
@@ -1,15 +1,41 @@
 package com.trivialTrivia.api;
```
Comment (esajdak) [inline src/main/test/com/trivialTrivia/api/TriviaApiTest.java]: Can we move this to match the other test structure test/java/com/trivialTrivia/api ?
I think it's causing an error when I pull this branch it- (I believe this is the cause.. if not feel free to help me out!)
image

image

File: src/main/java/com/trivialTrivia/entity/Category.java (hunk 1 above)
Comment (cruzang000) [reply src/main/java/com/trivialTrivia/entity/Category.java:7]: i added the asterisk down to one import line

File: src/main/test/com/trivialTrivia/api/TriviaApiTest.java (hunk 2 above)
Comment (cruzang000) [reply src/main/test/com/trivialTrivia/api/TriviaApiTest.java]: didnt even notice this, i moved the package over to the java directory

**Pull request merged**